src/vm/core_test.zig

```diff
@@ -1160,303 +1247,291 @@ test "set get value in vm memory" {
 }
 
 test "compute res op1 works" {
-    // ************************************************************
-    // *                 SETUP TEST CONTEXT                       *
-    // ************************************************************
-    // Initialize an allocator.
-    var allocator = std.testing.allocator;
-    var instruction = Instruction{
-        .off_0 = 0,
-        .off_1 = 1,
-        .off_2 = 2,
-        .dst_reg = instructions.Register.AP,
-        .op_0_reg = instructions.Register.AP,
-        .op_1_addr = instructions.Op1Src.AP,
-        .res_logic = instructions.ResLogic.Op1,
-        .pc_update = instructions.PcUpdate.Regular,
-        .ap_update = instructions.ApUpdate.Regular,
-        .fp_update = instructions.FpUpdate.Regular,
-        .opcode = instructions.Opcode.NOp,
-    };
-
     // Create a new VM instance.
-    var vm = try CairoVM.init(allocator, .{});
+    var vm = try CairoVM.init(
+        std.testing.allocator,
+        .{},
+    );
     defer vm.deinit();
-
     vm.run_context.ap.* = Relocatable.new(1, 0);
-    // ************************************************************
-    // *                      TEST BODY                           *
-    // ************************************************************
-
-    const value_op0 = relocatable.fromFelt(starknet_felt.Felt252.fromInteger(2));
-    const value_op1 = relocatable.fromFelt(starknet_felt.Felt252.fromInteger(3));
-
-    const actual_res = try computeRes(&instruction, value_op0, value_op1);
-    const expected_res = value_op1;
 
-    // ************************************************************
-    // *                      TEST CHECKS                         *
-    // ************************************************************
     try expectEqual(
```

Comment:
Is this more readable in your opinion? Or it's more Zig style to put it in the `try`? I personally feel that having `expected_value` and `actual_value` defined is very convenient, but maybe less lines is better?